src/libponyrt/options/options.c

```diff
@@ -214,6 +223,11 @@ int ponyint_opt_next(opt_state_t* s)
     printf("%s: '%s' option is ambiguous!\n", s->argv[0], s->argv[s->idx]);
     return -2;
   }
+  else if(m == (opt_arg_t*)2)
+  {
+    printf("%s: Unrecognised pony runtime option: '%s'!\nRun '%s --ponyhelp' for available pony runtime options.\n", s->argv[0], s->argv[s->idx], s->argv[0]);
```

Comment:
@SeanTAllen does this revised error message sound good?

```suggestion
    printf("%s: Unrecognised pony runtime option: '%s'!\nRun '%s --ponyhelp' for available pony runtime options.\nIf you're trying to use this as an application argument, you'll need to rename it to not include the '--pony' prefix.\n", s->argv[0], s->argv[s->idx], s->argv[0]);
```

Reply:
Yes. Looks good to me.